yt/data_objects/level_sets/api.py

```diff
@@ -19,9 +19,7 @@
 from .clump_handling import \
     Clump, \
     find_clumps, \
-    get_lowest_clumps, \
-    write_clump_index, \
-    write_clumps
```

Comment:
Good question. Firstly, `write_clumps` and `write_clump_index` are two variations if saving clump information to a text file. The difference is that `write_clumps` saves only the leaf nodes, whereas `write_clump_index` saves the full hierarchy denoting levels with indentation. 

Both of these functions have been deprecated since yt 3.3 and were removed from the narrative docs and cookbook recipes sometime before yt 3.4.